docs/en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md

```diff
@@ -199,6 +199,72 @@ convention in GraphQL APIs to use lowerCamelCase fields, so this is given by def
 
 [/notice]
 
+### Bulk loading models
+
+It's likely that in your application, you have a whole collection of classes you want exposed to the API, with roughly
+the same fields and operations exposed on them. It can be really tedious to write a new declaration for every single
+dataobject in your project, and as you add new ones, there's a bit of overhead in remembering to add it to the
+GraphQL schema.
+
+Common use cases might be:
+
+* Add everything in `App\Models`
+* Add every implementation of `BaseElement`
+* Add anything with the `Versioned` extension
+* Add everything that matches `src/*Model.php`
+
+You can create logic like this using the `bulkLoad` configuration file, which allows you to specify groups of directives
```

Comment:
> You can create logic like this using the `bulkLoad` configuration file

I think it would be good to clarify here that this means either in `app/_graphql/schema.yml` or `app/_graphql/bulkLoad/bulkLoad.yml`. I’ve been using v4 beta and I wasn’t sure where this file/config should go until I saw the updates to the “Configuring your schema” doc.

Either that, or just include an example `bulkLoad` config below this paragraph and include the file path above it.